### ISSUE TITLE
Xbox 360 controller: 'Guide' button not working

### ISSUE THREAD

**Issue by bycrean** [libretro/ludo#445]
When entering the menu after having started a game, the UI hints at the buttom indicate that pressing the 'Guide' button on the controller resumes the game. However, pressing that button has no effect.

**Comment by johnnyq** [comment]
Two Things @bycrean 

1. Are you using the controller wireless or wired
2. Are you using an aftermarket Microsoft 360 controller

See here for further information on Joypads

https://jean-andre-santoni.gitbook.io/ludo/input-methods

**Comment by bycrean** [comment]
@johnnyq

I use a wired, original Xbox 360 controller.

**Comment by johnnyq** [comment]
@bycrean Thank you that information should help @kivutar to diagnose. I don't have an xbox 360 controller or a Windows 10 computer to try.

**Comment by kivutar** [comment]
Yep, we can't get it to work on Windows sadly. GLFW simply doesn't expose this button.

It does work on OSX and Linux.

It's because Windows itself uses this guide button to spawn the XBox Gaming app on Windows. And there is nothing we can do about it.

**Comment by bycrean** [comment]
@kivutar I understand. But if it's not working on Windows, then the controller icon in the UI of the Windows version of Ludo should reflect that and not highlight the 'Guide' button but L3+R3. Otherwise, users are shown to press the 'Guide' button and are then confused that nothing happens (as I was).
